app/src/components/SignUpForm.js

```diff
@@ -0,0 +1,109 @@
+import React, {Component} from 'react';
+import {Validations} from '../common/Validations.js';
+import UsersDataRequests from '../common/UsersDataRequests';
+import '../../node_modules/bootstrap/dist/css/bootstrap.min.css';
+import '../css/App.css';
+import {connect} from 'react-redux';
+import '../../node_modules/jquery/dist/jquery.min'
+import UserForm from './UserForm';
+import isGuest from "../common/isGuest";
+import endPointsList from "../common/endPointsList";
+import {NavLink} from 'react-router-dom'
+import Alerts from './Alerts';
+
+class SignUpForm extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {};
+
+        this.signUpInputId = {
+            firstName: 'sign-up-firstName',
+            lastName: 'sign-up-lastName',
+            email: 'sign-up-email',
+            genderMale: 'sign-up-gender-male',
+            genderFemale: 'sign-up-gender-female',
+            age: 'sign-up-age',
+            photo: 'sign-up-photo',
+            middleName: 'sign-up-middleName'
+        };
+        this.handleSubmit = this.handleSubmit.bind(this);
+    }
+
+    async handleSubmit(event) {
+        event.preventDefault();
+        const validateFlag = this.validateForm();
+        if (validateFlag) {
+            console.log('valid validation');
```

Comment:
remove all console.log from the repository. No console.log in the 'production' code.